README.md

```diff
@@ -48,6 +48,10 @@ There are two UNIX domain sockets used by the node-driver-registrar:
   `/var/lib/kubelet/plugins/<drivername.example.com>/csi.sock). Note this is NOT
   the path to the registration socket.
 
+### Required volume mounts
+* `hostPath` volume to the directory containing the kubelet registration socket (typically in `/var/lib/kubelet/plugins_registry`). Must be mounted at `/registration`. The host path must match the directory path in `--kubelet-registration-path` flag.
```

Comment:
Can this go under the registration socket section above?  Also the hostpath should be the kubelet plugin registration directory, not the kubelet-registration-path, which is poorly named.